src/main/cc/wfa/virtual_people/common/field_filter/field_filter_test.cc

```diff
@@ -0,0 +1,123 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "wfa/virtual_people/common/field_filter/field_filter.h"
+
+#include "absl/status/status.h"
+#include "gmock/gmock.h"
+#include "google/protobuf/text_format.h"
+#include "gtest/gtest.h"
+#include "src/main/cc/wfa/virtual_people/common/field_filter/test/test.pb.h"
+#include "src/test/cc/testutil/matchers.h"
+#include "src/test/cc/testutil/status_macros.h"
+
+namespace wfa_virtual_people {
+namespace {
+
+using ::wfa::StatusIs;
+using ::wfa_virtual_people::test::TestProto;
+
+TEST(FieldFilterTest, FromMessage_FloatNotSupported) {
```

Comment:
Please don't use underscores in test names.

Reply:
Done.